docs/guide/users.md

```diff
@@ -0,0 +1,54 @@
+# Users and user roles
+
+A **"user"** in Apostrophe is an account (usually for a human or benevolent robot) that can be used to log into the website. Beyond that, users get specific sets of permissions based on the **role** they are assigned. More on that to come.
+
+## Adding new users
+
+If you want to do anything with your Apostrophe website, you will need users. There are two main ways to add them: using the command-line task or through the user interface.
+
+### Adding users with the CLI task
+
+The CLI task for adding users is primarily useful to add the very first admin user. The command is:
+
+```bash
+node app @apostrophecms/user:add my-user admin
+```
+
+The two final arguments passed to the task are, in order, the new username (`my-user`) and the user role (`admin`).
```

Comment:
I think we should role right out of the gate with a user table. Here might be good. Or maybe even above.

Reply:
I don't think roles should come before adding new users. And it's a lot to put in the middle of what are otherwise short sections. I'll add links down to the roles section.